technical-reports/format/groups.md

```diff
@@ -127,8 +127,8 @@ For example:
 Suggested ways tools MAY use this property are:
 
 - A style guide generator could render a section for each group and use the description as an introductory paragraph
-- A GUI tool that lets users browse or select tokens could display this info alongside the corresponding group or as a tooltip
```

Comment:
I would leave GUI as it is here. This advice applies to any tool that has a graphical user interface, regardless of whether it is a (UI) design tool or not.